app/src/main/java/org/stepic/droid/ui/dialogs/LoadingProgressDialog.kt

```diff
@@ -0,0 +1,26 @@
+package org.stepic.droid.ui.dialogs
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.support.annotation.StringRes
+import android.view.Window
+import kotlinx.android.synthetic.main.dialog_progress.*
+
+import org.stepic.droid.R
+
+
+class LoadingProgressDialog
+@JvmOverloads constructor(
+        context: Context,
+        @StringRes private val titleRes: Int = R.string.loading) : Dialog(context) {
```

Comment:
What if we change orientation, `LoadingProgressDialog`will dismissed? (I think expected is not)

Reply:
May be using of `DialogFragment` is better idea here?

Reply:
This class replaces old `LoadingProgressDialog` and provides the same logic. The only change is the custom design.

Reply:
There is a `LoadingProgressDialogFragment` for that propose.